# HG changeset patch
# User Rik <rik@octave.org>
# Date 1549507819 28800
#      Wed Feb 06 18:50:19 2019 -0800
# Branch stable
# Node ID 736f26d34f79087e8b15327b54598e4fc8ef96b1
# Parent  6764d6a428caf1ea4f3cab1454c951f5ecf18b60
qmr.m: Fix mistaken call to "print usage" without underscore (bug #55599).

* qmr.m: Add missing '_' so that print_usage () is called in input validation.

diff --git a/scripts/sparse/qmr.m b/scripts/sparse/qmr.m
--- a/scripts/sparse/qmr.m
+++ b/scripts/sparse/qmr.m
@@ -257,17 +257,17 @@ function [x, flag, relres, iter, resvec]
         printf ("The iterate returned (number %i) ", iter);
         printf ("has relative residual %e\n", res1);
       endif
     elseif (nargout < 2)
       printf ("qmr converged at iteration %i ", iter);
       printf ("to a solution with relative residual %e\n", res1);
     endif
   else
-    print usage();
+    print_usage();
   endif
 
 endfunction
 
 
 %!demo
 %! % Solve system of A*x=b
 %! A = [5 -1 3;-1 2 -2;3 -2 3];
